front/app/containers/IdeasShow/components/ProposalInfo/index.tsx

```diff
@@ -45,21 +46,16 @@ const ProposalInfo = ({ idea, compact }: Props) => {
 
   const { code } = ideaStatus.data.attributes;
 
+  // All the codes checked for here are ProposalsStatusCode types (see front/app/api/idea_statuses/types.ts)
   const showCountDown =
     code === 'proposed' || code === 'expired' || code === 'custom';
-
   const showProgressBar =
     code === 'proposed' ||
     code === 'threshold_reached' ||
     code === 'custom' ||
     code === 'ineligible' ||
     code === 'answered';
-
-  const showVoteButtons =
-    code === 'proposed' ||
-    code === 'threshold_reached' ||
-    code === 'custom' ||
-    code === 'answered';
+  const showVoteButtons = showProposalsReactions(idea.data);
```

Comment:
This is a behavior change for admins as well. The vote button for proposals will always show for admins now (including when status code is `expired` or `ineligible`). We will check with product whether this is fine.

The button visibility will be fixed for users who cannot manage the project though.

Reply:
(If we want to change this, changes will most likely need to happen in the BE, so you can already review this.)